homeassistant/components/aladdin_connect/sensor.py

```diff
@@ -105,6 +117,7 @@ def device_info(self) -> DeviceInfo | None:
             identifiers={(DOMAIN, self._device_id)},
             name=self._name,
             manufacturer="Overhead Door",
+            model=f"Model: {self._model}",
```

Comment:
The "Model" was added to make the device info more clear.  Without it, it only shows "02", which didn't make sense to me.  Otherwise it's real need/meaning.